tests/components/epion/test_config_flow.py

```diff
@@ -0,0 +1,78 @@
+"""Tests for the Epion config flow."""
+from unittest.mock import Mock, patch
+
+from epion import EpionAuthenticationError, EpionConnectionError
+import pytest
+
+from homeassistant import data_entry_flow
+from homeassistant.components.epion.const import DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.const import CONF_API_KEY
+from homeassistant.core import HomeAssistant
+
+API_KEY = "test-key-123"
+
+
+@pytest.fixture(name="epion_api")
+def mock_controller():
+    """Mock the Epion API."""
+    api = Mock()
+    with patch("epion.Epion", return_value=api):
+        yield api
+
+
+async def test_user_flow(hass: HomeAssistant, epion_api: Mock) -> None:
```

Comment:
Can we split these tests up? 1 test for full flow and a parametrized test for the errors.

Reply:
And now also one for the duplicated entry check